XcodeServerSDK/Utils/JSON.swift

```diff
@@ -15,34 +15,32 @@ public protocol JSONSerializable {
 
 public class JSON {
 
-    private class func parseDictionary(data: NSData) -> ([String: AnyObject]!, NSError!) {
-
-        let (object: AnyObject!, error) = self.parse(data)
-        return (object as! [String: AnyObject]!, error)
+    private class func parseDictionary(data: NSData) -> ([String: AnyObject]?, NSError?) {
+        let (object, error) = self.parse(data)
+        return (object as? [String: AnyObject], error)
     }
 
     private class func parseArray(data: NSData) -> ([AnyObject]!, NSError!) {
-        
-        let (object: AnyObject!, error) = self.parse(data)
-        return (object as! [AnyObject]!, error)
+        let (object, error) = self.parse(data)
+        return (object as? [AnyObject], error)
     }
     
-    public class func parse(url: NSURL) -> (AnyObject!, NSError!) {
+    public class func parse(url: NSURL) -> (AnyObject?, NSError?) {
         
-        var error: NSError?
-        if let data = NSData(contentsOfURL: url, options: NSDataReadingOptions.allZeros, error: &error) {
+        do {
+            let data = try NSData(contentsOfURL: url, options: NSDataReadingOptions(rawValue: 0))
```

Comment:
:+1: Nice.

Just btw, can you put the options back to `NSDataReadingOptions.allZeros`?